guides/common/modules/ref_example-playbooks-based-on-modules-from-project-ansible-collection.adoc

```diff
@@ -0,0 +1,153 @@
+[id="example-playbooks-based-on-modules-from-{project-context}-ansible-collection"]
+= Example playbooks based on modules from {Project} Ansible Collection
+
+All playbooks based on modules from {Project} Ansible Collection must include parameters detailing how to connect to the {Project} API.
+The following examples use Ansible vault and module defaults group to provide these parameters, and they authenticate using a username and password.
+For more information, see xref:creating-a-playbook-with-modules-from-{project-context}-ansible-collection[].
+
+.Example Ansible playbook: Enable repositories and create a content view
+====
+This example playbook uses the following modules:
+
+* `{ansible-namespace}.repository_set`
+* `{ansible-namespace}.content_view`
```

Comment:
you also use `repository_sync` -- it's technically not necessary for creating the data structures, so I'd probably avoid it.

Reply:
repository_sync removed from this example